<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-VLHT10O" failures="3" tests="9" name="Surefire test" time="18.035" errors="0" timestamp="2024-09-02T10:20:42 PDT">
  <testcase classname="api.Test.PetTests" name="Post" time="5.166">
    <failure type="java.lang.AssertionError" message="expected [200] but found [405]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [405]
at api.Test.PetTests.Post(PetTests.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- Post -->
  <testcase classname="api.Test.UserPropertiesTest" name="Create" time="1.783"/>
  <testcase classname="api.Test.UserPropertiesTest" name="Get" time="1.198">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.Test.UserPropertiesTest.Get(UserPropertiesTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- Get -->
  <testcase classname="api.Test.UserPropertiesTest" name="Update" time="2.325"/>
  <testcase classname="api.Test.UserPropertiesTest" name="Delete" time="1.157">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.Test.UserPropertiesTest.Delete(UserPropertiesTest.java:82)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 37 stack frames]]>
    </failure>
  </testcase> <!-- Delete -->
  <testcase classname="api.Test.UserTests" name="Post" time="1.153"/>
  <testcase classname="api.Test.UserTests" name="Get" time="1.18"/>
  <testcase classname="api.Test.UserTests" name="Update" time="2.356"/>
  <testcase classname="api.Test.UserTests" name="Delete" time="1.137"/>
</testsuite> <!-- Surefire test -->
